app/components/Button.tsx

```diff
@@ -111,7 +111,7 @@ const RealButton = styled.button<{
       }
 
       &:disabled {
-        background: none;
+        background: ${lighten(0.05, props.theme.danger)};
```

Comment:
this was displaying as white! I'm a bit surprised this wasn't clearly an issue elsewhere but I guess disabled danger buttons are rare enough

Reply:
If something dangerous is disabled then it's usually hidden I suppose!